### PR TITLE
Update pvc storage name and size

### LINKED ISSUE #7829: Updating metering configuration storage class or size doesn't automatically update the metering PVC
Updating metering configuration storage class or size doesn't automatically update the metering PVC.
We should probably mention that in the documentation and describe how the admin can do that in the seed cluster manually if needed.

Also, we can show those params disable after initial setup.

### PR DESCRIPTION
**Pull request by moadqassem**
**What this PR does / why we need it**:
Updating the metering pvc storage name and size. 
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7829

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: pkg/ee/metering/pvc.go [hunk 1]
```diff
@@ -52,29 +51,43 @@ func persistentVolumeClaimCreator(ctx context.Context, client ctrlruntimeclient.
...
+
+func updatePVCStorageSizeAndName(pvc *corev1.PersistentVolumeClaim, seed *kubermaticv1.Seed) error {
+	pvc.Spec.StorageClassName = pointer.StringPtr(seed.Spec.Metering.StorageClassName)
+	pvc.Spec.AccessModes = []corev1.PersistentVolumeAccessMode{
+		corev1.ReadWriteOnce,
+	}
+
+	pvcStorageSize, err := resource.ParseQuantity(seed.Spec.Metering.StorageSize)
+	if err != nil {
+		return fmt.Errorf("failed to parse value of metering pvc storage size %q: %v", "100Gi", err)
```
Comment (simon-wessel) [inline pkg/ee/metering/pvc.go]: Was it intended to include `seed.Spec.Metering.StorageSize` instead of the hard coded `"100Gi"`?

Comment (moadqassem) [reply pkg/ee/metering/pvc.go]: This is actually a copy and past

**Pull request merged**